### DISCUSSION TITLE
Button to give night vision/full bright vision

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#46] [Feedback]
A button for nightvision! 

It's quite hard to edit and create in caves or underground, being able to switch of all the darkness to see better would improve the expirience!

![Nightvision](https://user-images.githubusercontent.com/16018643/223910689-d66c3235-94d9-4f8e-841f-ca254dd4ed59.png)

**Comment by ihategravel2** [comment]
I think a "full bright" affect would be better then night vision.

**Reply by Landy005** [reply]
That's what I meant, but full bright be more clear, thanks!

**Comment by minecraftchloe** [comment]
Absolutely, yes! Agreed!
